src/transform/ThemeTransform.css

```diff
@@ -35,51 +33,71 @@
   background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsMAAA7DAcdvqGQAAAAadEVYdFNvZnR3YXJlAFBhaW50Lk5FVCB2My41LjEwMPRyoQAAAGJJREFUKFN1jdENgCAMBYmJn47Bak7DZrhTpc/XIm34OAjXA4qIgHI/dSBbLGTcOKjBryFlinGmjDQGiOF0MQkxI3v5wq6L38qR7SnsAx8ul37igPjAd+o5Oz2MRA+xY4ZSXuaW6wYouOLpAAAAAElFTkSuQmCC);
 }
 
-.pagelib-theme-dark .content table.infobox,
-.pagelib-theme-dark .content table.infobox th {
-  /* The infobox background. Some backgrounds need important to override inline styles like
-     https://en.m.wikipedia.org/wiki/Wolverine_(character)?oldid=789987909.
-     https://phabricator.wikimedia.org/diffusion/SMIN/browse/master/resources/skins.minerva.content.styles/hacks.less;2cc9516cde6027234de9881d503ed7161e0e2ace$44 */
+.pagelib-theme-dark .content table,
+.pagelib-theme-dark .content td,
+.pagelib-theme-dark .content tr,
+.pagelib-theme-dark .content th {
   background: #27292d !important;
+  color: #f8f9fa !important;
 }
-
-.pagelib-theme-sepia .content table.infobox,
-.pagelib-theme-sepia .content table.infobox th {
+.pagelib-theme-sepia .content table,
+.pagelib-theme-sepia .content td,
+.pagelib-theme-sepia .content tr,
+.pagelib-theme-sepia .content th {
   background: #f0e6d6 !important;
+  color: #222 !important; 
 }
 
-/* Same specificity as in MobileFrontend/less/content/hacks.less */
-.pagelib-theme-dark .content table.infobox tr {
-  /* The infobox row background. Some backgrounds need important to override inline styles like
-     https://en.m.wikipedia.org/wiki/Wolverine_(character)?oldid=789987909. */
-  background: #27292d !important;
+.pagelib-theme-dark .content div,
+.pagelib-theme-dark .content span,
+.pagelib-theme-dark .content ul,
+.pagelib-theme-dark .content li {
+  background: transparent !important;
+  color: #f8f9fa !important;
+}
+.pagelib-theme-sepia .content div,
+.pagelib-theme-sepia .content span,
+.pagelib-theme-sepia .content ul,
+.pagelib-theme-sepia .content li {
+  background: transparent !important;
+  color: #222 !important; 
 }
 
-.pagelib-theme-sepia .content table.infobox tr {
+.pagelib-theme-dark .content div.quotebox {
+  background: #27292d !important;
+}
+.pagelib-theme-sepia .content div.quotebox {
   background: #f0e6d6 !important;
 }
 
-.pagelib-theme-dark li.gallerybox div.thumb {
-  border-color: #43464a;
-  background: #27292d;
+.pagelib-theme-dark .content * {
```

Comment:
Agreed these are _usually_ not recommended - it's a blunt tool.

The reason I switched to this 'baseline' approach:

There were many places, especially in non-EN wikis, where inline styles are used - often on elements having _no css classes_ specified. Without having CSS classes, it was almost impossible to identify and target such elements for theming.

This approach handles such elements and provide consistent theming.

The fallout from this approach:
- simplified other theme transform CSS - much of it no longer needed
- theme CSS needs to come before other transform CSS in the file delivered by the package (otherwise the other transforms wouldn't be able to override theme CSS if they needed to - note when doing so these transforms need to target elements using a >= level of specificity as used by the theme transform)